### ISSUE TITLE
v2.2.1 is not available on Firefox add-ons

### ISSUE THREAD

**Issue by shahin365** [easonwong-de/Adaptive-Tab-Bar-Colour#102]
**2.1.4** is the latest version: https://addons.mozilla.org/en-US/firefox/addon/adaptive-tab-bar-colour/

**Comment by easonwong-de** [comment]
Yeah, we're still waiting for it to be reviewed by Mozilla.

**Comment by NotoriousNico** [comment]
Mozilla sure is taking its sweet time with approving the new version. 😅

**Comment by OutshineIssue** [comment]
I guess this issue can be closed now! 😉

![image](https://github.com/user-attachments/assets/0ba2bd09-d04a-42d5-af05-34f9f15a33f4)

**Comment by easonwong-de** [comment]
Yup… although I pulled down the update because there’s a bug with v2.2.1. Now we are waiting for v2.2.2 :(

**Comment by loxoron218** [comment]
Just curious. How long does the _mozilla_ reviewing process take?

**Comment by NotoriousNico** [comment]
> Yup… although I pulled down the update because there’s a bug with v2.2.1. Now we are waiting for v2.2.3 :(

For what it's worth:
I was affected by the bug in 2.2.1 and my settings didn't get saved. Uninstalling and reinstalling the extension fixed the issue for me.

This information might be helpful for others who have the same issue and are looking for a workaround.
